observer/test/observer.test.js

```diff
@@ -0,0 +1,64 @@
+import assert from 'node:assert'
+import { observeScheduledRewards } from '../lib/observer.js'
+import { getPgPools } from '@filecoin-station/spark-stats-db'
+
+describe('observer', () => {
+  describe('observeScheduledRewards', () => {
+    let pgPools
+
+    before(async () => {
+      pgPools = await getPgPools()
+    })
+
+    it('observes scheduled rewards', async () => {
+      await pgPools.evaluate.query('DELETE FROM daily_participants')
+      await pgPools.evaluate.query('DELETE FROM participants')
```

Comment:
Let's move these DB cleanups to a `beforeEach` hook, please.

Tests must not reuse data from other tests, otherwise `it.only` and `mocha -g "<testname>"` won't work.